web/src/index.css

```diff
@@ -8,4 +8,19 @@ body {
 
 code {
     font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
+}
+
+.react-resizable-handle {
+    position: absolute;
+    right: -5px;
+    bottom: 0;
+    z-index: 1;
+    width: 10px;
+    height: 100%;
+    cursor: col-resize;
+}
+
+.react-resizable {
+    position: relative;
+    background-clip: padding-box;
```

Comment:
Please just pass these in to the component that uses them since they aren't truly global and we always pass the antd class modifications in to the component directly

Reply:
@KaWaite , moved to resizable component.